### DISCUSSION TITLE
Feed Generation

### DISCUSSION THREAD

**Discussion by JakeJakoubek** [bluesky-social/atproto#1936] [How to do it?] [answered]
We’re attempting to build a Feed Generator and are following instructions in https://github.com/bluesky-social/feed-generator/blob/7421e718fc8516570a16284f9dc49261eba6d4c8/README.md. It says to edit **setup.json**. Where may we find **setup.json**? Who may we contact with any other questions as we fumble forward with this?

**Comment by bnewbold** [comment, marked as answer]
Hi @JakeJakoubek,

I'm not sure why, but you seem to have linked to a 7-month-old copy of the `feed-generator` README. The current version of the README ([link](https://github.com/bluesky-social/feed-generator/blob/main/README.md)) has no mention of `setup.json`.
